tests/support_server/configure.pm

```diff
@@ -49,6 +57,43 @@ sub run {
 
     script_output($script);
 }
+sub _firewall {
+    assert_script_run 'sysctl net.ipv4.ip_forward';
+}
+sub _packages {
+    #select_console 'root-console';
+    
+    assert_script_run "zypper -n install openssh tftp dhcp-server bind yast2-iscsi-lio-server";
+    assert_script_run "rpm -q openssh";
+    assert_script_run "rpm -q tftp";
+    assert_script_run "rpm -q dhcp-server";
+    assert_script_run "rpm -q bind";
+    assert_script_run "rpm -q yast2-iscsi-lio-server";
```

Comment:
Just a nitpicking, but could you please change the double quotes to single quotes in cases when you do not need the strings to be interpolated.